### ISSUE TITLE
Custom playlists UI overlap with a different module

### ISSUE THREAD

**Issue by lucastbelem** [themeetinghouse/web#709]
@JonEsparaz I added the playlist type that you created last week for us into the Easter page (themeetinghouse.com/easter). I added it below the locations module and the issue seen in the image below appears.

@deeinstereo is that where you'd like the playlist to be? If so, @JonEsparaz anything you can do to clean that up? If we're moving it, maybe not an urgent issue.

Also, @JonEsparaz, I noticed that this playlist goes oldest-newest, but most of our playlists go newest-oldest. Although, I wanted to check this assumption and noticed that other playlists also go oldest-newest. @deeinstereo is there a default for this? Would it be helpful if we could pick that in the json?

**Comment by lucastbelem** [comment]
<img width="1433" alt="Screen Shot 2021-03-22 at 10 58 33 AM" src="https://user-images.githubusercontent.com/48295718/112010868-13983300-8afe-11eb-8c0a-7a5d95902c7b.png">

**Comment by deeinstereo** [comment]
@lucastbelem that looks like a Good Spot! @JonEsparaz can you fix the header for that playlist? Also is it a custom title? 
And please make it go newest to oldest, and let's get rid of a bunch of them. Where can I find it @lucastbelem to customize what's included?

**Comment by lucastbelem** [comment]
@deeinstereo title is in the easton.json file in Visual Code. As for removing videos, I think it can only be done in Dynamo directly (is that true, @JonEsparaz?) If so, then @deeinstereo just tell me which videos you want removed and I'll do it.

**Comment by JonEsparaz** [comment]
> @deeinstereo title is in the easton.json file in Visual Code. As for removing videos, I think it can only be done in Dynamo directly (is that true, @JonEsparaz?) If so, then @deeinstereo just tell me which videos you want removed and I'll do it.

As recently mentioned, there has always been a way to remove videos from custom playlists: https://github.com/themeetinghouse/web/issues/694#issuecomment-791013543

**Comment by JonEsparaz** [comment]
I'll add a sort order control and fix the styling issue.

Of note, this is related to https://github.com/themeetinghouse/web/issues/670 as this problem is being caused by inappropriate use of absolute positioning.

**Comment by lucastbelem** [comment]
> > @deeinstereo title is in the easton.json file in Visual Code. As for removing videos, I think it can only be done in Dynamo directly (is that true, @JonEsparaz?) If so, then @deeinstereo just tell me which videos you want removed and I'll do it.
> 
> As recently mentioned, there has always been a way to remove videos from custom playlists: [#694 (comment)](https://github.com/themeetinghouse/web/issues/694#issuecomment-791013543)

Thanks Jon! I had forgotten about this!

**Comment by JonEsparaz** [comment]
@lucastbelem why'd you move the component? Is this fix no longer needed?

**Comment by lucastbelem** [comment]
@JonEsparaz David asked me to move it...maybe didn't realize you were getting on it right away. @deeinstereo good if we move it back?

**Comment by JonEsparaz** [comment]
#711 will move it back below the location selector. If that's where you want it, no need to change anything.

**Comment by deeinstereo** [comment]
@lucastbelem yeah if it's fixed let's move it back! @JonEsparaz how are we on organizing it newest to oldest. Those old videos are killing me :P

**Comment by lucastbelem** [comment]
@deeinstereo Jon fixed this already and made it possible for us to re-order them. @GeorgeBellTMH just has to merge it. Should be good to go tonight.

**Comment by JonEsparaz** [comment]
The JSON will look like this, but as @lucastbelem said, waiting on the merge and deploy for this to work.

```json
{
  "type": "list",
  "style": "horizontal",
  "class": "custom-playlist",
  "playlist": "Easter",
  "header1": "Easters 2008-2020",
  "sortOrder": "DESC",
  "margin": {
    "marginTop": 24
  }
},
```

**Comment by deeinstereo** [comment]
Thanks! sorry I'm late to the party!
